src/github.com/stellar/gateway/bridge/config/config.go

```diff
@@ -68,6 +68,15 @@ func (c *Config) Validate() (err error) {
 		return
 	}
 
+	for _, asset := range c.Assets {
+		if asset.Issuer == "" {
+			if asset.Code != "XLM" {
+				err = errors.New("Issuer param is required for "+asset.Code)
+				return
+			}
+		}
```

Comment:
Yes. the previous version didnt have asset validation. 
Will see if I can add that to this PR